### PR TITLE
TLC-120-2: Offline/Dashboard/Report improvements, add Immuns. MR

### PR DESCRIPTION
**Pull request by ouiliam**

### CODE REVIEW

**Comment by ouiliam** [comment]
I planned on deleting the previous unmerged PR, but not sure if it contains some commits which are not in this one?

File: configuration/scripts/post_start/copyOfflineConcepts.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def execute() :
         insertConceptToOfflineConcepts(concepts[index], uuid, name)
 
 def getTopLevelParents() :
-    query = "select DISTINCT concept_id from concept_name where name = 'All Observation Templates' or name = 'Registration Offline Concepts' or name = 'Diagnosis Set of Sets' or name = 'Visit Diagnoses'"
```
Comment (rbuisson) [inline configuration/scripts/post_start/copyOfflineConcepts.py:117]: @ouiliam here we don't sync the "All Observation Templates" concepts anymore?

Comment (ouiliam) [reply configuration/scripts/post_start/copyOfflineConcepts.py:117]: there is now an offline consultation form (without image upload), i copied all the forms for offline into registration offline concepts. this way the image concept wont be included

Comment (ouiliam) [reply configuration/scripts/post_start/copyOfflineConcepts.py:117]: @rbuisson

**Pull request merged**